tidb-cloud/tidb-cloud-console-auditing.md

```diff
@@ -0,0 +1,145 @@
+| AuditEventCreateImportTask | Create an import task |
+| AuditEventCancleImportTask | Cancel an import task |
+| AuditEventExitImportTask | Exit an import task |
+| AuditEventCreateCluster | Create a cluster |
+| AuditEventDeleteCluster | Delete a cluster |
+| AuditEventScaleCluster | Scale a cluster |
+| AuditEventCreateBackup | Make a backup |
+| AuditEventDeleteBackup | Delete a backup |
+| AuditEventRestoreBackup | Restore from a backup |
+| AuditEventUpdateAuditLogStatus | Enable / disable database audit logging |
+| AuditEventCreateAuditLogAccessRecord | Add database audit log filter conditions |
+| AuditEventDeleteAuditLogAccessRecord | Delete database audit log filter conditions |
+| AuditEventUpdateUserRole | Modify user roles |
+
+## Audit log storage policy
+
+- The audit log information is stored in AWS ES.
+- The storage time is 90 days, after which the audit logs will be automatically cleaned up.
+
+## View audit logs
+
+The Console audit log information is temporarily accessible only to internal personnel of TiDB Cloud. If you need to view audit log information, contact PingCAP support for it.
+
+## Audit log fields
+
+The fields in the audit log include basic fields and extended fields.
+
+The basic fields are as follows:
+
+| Field name | Data type | Description |
+|---|---|---|
+| timestamp | timestamp | Time of event |
+| Auditeventtype | string | Event type |
+| userID | uint64 | User ID |
+| clientIP | string | Client IP |
+| isSuccess | bool | Event result |
+
+Extended fields supplement the description information of events based on different event types to ensure the integrity and availability of audit information
+
+| Audit event type | Extended fields | Data type | Description |
+|---|---|---|---|
+| AuditEventSignIn |  |  |  |
+| AuditEventSignOut |  |  |  |
+| AuditEventUpdateUserProfile |  |  |  |
+| AuditEventUpdateMFA | enableMFA | bool |Enable / disable MFA |
+| AuditEventCreateProject | projectName | string | Project name |
+| AuditEventUpdateProject | oldProjectName <br/> newProjectName | string <br/> string | Old Project name <br/> New project name |
+| AuditEventDeleteProject | projectName | string | Project name |
+| AuditEventInviteUserIntoProject | email <br/> Role | string <br/> string | Email <br/> Role name |
+| AuditEventDeleteProjectUser | email <br/> Role | string <br/> string | Email <br/> Role name |
+| AuditEventUpdateOrg | orgName <br/> timeZone | string <br/> uint| Organization name <br/> Time zone |
+| AuditEventCreateIntegration | integrationType | string | Integration type |
+| AuditEventDeleteIntegration | integrationType | string | Integration type |
+| AuditEventListOrgUsers |  |  |  |
+| AuditEventListProjectUsers |  |  |  |
+| AuditEventAddNewPaymentMethod | cardNumber | string | Payment card number (desensitized) |
+| AuditEventUpdatePaymentMethod | cardNumber | string | Payment card number (desensitized) |
+| AuditEventDeletePaymentMethod |  |  |  |
```

Comment:
```suggestion
| AuditEventDeletePaymentMethod |  |  | Currently not recorded |
```

Reply:
This is the place to write the field description, it is not appropriate to add such information. And it is not that there is no record, the record information is incomplete.